iscsi/initiator.go

```diff
@@ -166,6 +179,33 @@ func IsTargetLoggedIn(ip, target string, ne *util.NamespaceExecutor) bool {
 	return found
 }
 
+func manualScanSession(ip, target string, ne *util.NamespaceExecutor) {
+	opts := []string{
+		"-m", "node",
+		"-T", target,
+		"-p", ip,
+		"--rescan",
+	}
+	_, err := ne.ExecuteWithTimeout(sessionScanTimeout, iscsiBinary, opts)
```

Comment:
if we have to manual scan but it failed, should we error out?

Reply:
I don't want to error out because if anything went wrong (get scan mode or the actual manual re-scan), I want to fall back to auto scan and only print the error out.

Reply:
If the manual scan failed, is it possible for the auto scan to succeed?

Reply:
Yeah, because auto scan already happened before the calling to `getIscsiNodeSessionScanMode` and `manualScanSession` when we do the login at https://github.com/longhorn/go-iscsi-helper/blob/9fe20ee186fb931cbb9b70253ba4ffe434c17b46/iscsi/initiator.go#L103 if the system determine that we can auto rescan. 

If somehow my `getIscsiNodeSessionScanMode` goes wrong and says that we need to do a false manual rescan, `manual rescan` will be timeout (I don't know why it times out but it did timeout if when do a auto rescan and then manual rescan together). In this case, I don't want to prevent the volume from being attached to the pod.

Reply:
I fact, my first solution is much simpler than this one. I didn't check whether the node has `node.session.scan mode` set to `manual` or `auto`. I just tried to do manual rescan with a timeout in all cases. However, it is a problem for many users with `auto` mode because suddenly they have to wait for `scanTimeout` seconds for every volume attachment. It makes the Longhorn feel slow.  Therefore, I implement this way with the hope that it can keep the attaching process fast but it also safe because in case our attempt to manually scan fail, we fall back to auto with a  `scanTimeout` waiting.

Reply:
fixed :)